.pre-commit-config.yaml

```diff
@@ -48,7 +48,7 @@ repos:
             - id: verify-copyright
             - id: verify-alpha-spec
       - repo: https://github.com/rapidsai/dependency-file-generator
-        rev: v1.13.11
+        rev: v1.16.0
```

Comment:
Updating the `rapids-dependency-file-generator` pre-commit hook version isn't necessary for this change, but since I just made that release (https://github.com/rapidsai/dependency-file-generator/pull/115), thought it'd be helpful to also do that across these PRs fiddling with build isolation. So that if that new release broke something, we'll find out sooner.